misk/src/main/kotlin/misk/web/WebConfig.kt

```diff
@@ -223,3 +227,17 @@ data class CorsConfig(
   /** A comma separated list of HTTP headers that are allowed to be exposed on the client. */
   val exposedHeaders: Array<String> = arrayOf()
 )
+
+data class ConcurrencyLimiterConfig(
+  /** If true, disables automatic load shedding when degraded. */
+  val enabled: Boolean = true,
```

Comment:
nit: maybe make this `disabled` instead, to keep it consistent with the other variable to disable the rate limiter in the WebConfig?

i think its slightly confusing to have 2 config variables controlling this, but they're the inverse of each other haha